### PR TITLE
17.9 release notes

### PR DESCRIPTION
**Pull request by diana-macrometa**
- Added a folder and intro topic for release notes.
- Changed number ordering for other topics to accomodate.

### CODE REVIEW

**Comment by jamesm-macrometa** [comment]
Adding @astrotars as a reviewer to confirm metadata for RN 17.9 page (once I upload it).  Hang tight!

**Comment by jamesm-macrometa** [comment]
Adding @astrotars as a reviewer to confirm metadata for RN 17.9 page (once I upload it).  Hang tight!

**Comment by jamesm-macrometa** [comment]
@astrotars When you get the chance, can you do me a favor and check out the newly added file - [ReleaseNotes_17_9.md](https://github.com/Macrometacorp/docs/pull/61/commits/f498240b71572655f0b3d764c50b832031195408) - and let me know if I formatted it properly for Docusaurus?  Thanks!

**Comment by astrotars** [comment]
@jamesm-macrometa If you're trying to set a default page with the index file, we need to do a little work in `sidebars.js` to support defaults files (as far as I know). Another options would be to rename your `_index.md` file to `v17.9` and put notes there. It would then look like this:

```
├── _category_.json
└── v17.9.md
```

And the contents of `v17.9.md` would look like this:

```
---
sidebar_position: 1
title: v17.9
slug: v17.9
---

This section of the documentation contains information about the latest release of the Macrometa platform.
```

Screen Shot 2022-05-24 at 6 57 57 PM

Something I'd like to look into is properly versioning our docs. More [here](https://docusaurus.io/docs/versioning).

File: docs/release-notes/ReleaseNotes_17_9.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+This update provides fixes for customer-reported issues.
+
+## Fixed Issues
+
+* API-179 - Fixed issues with API server latency.
+* STRM-296 / STRM-160 / STRM-215 - Fixed an issue where a metadata error in streams caused duplicate messages.
+* DEPL-168 / STRM-274 / STRM-255 - A timeout issue occasionally caused probe failures in Zookeeper.
+* DB-1340 - 
+* DB-1442 - If you added documents to a collection from different regions, occasionally a document would fail to add after trying to use a unique key claimed by another document.
```
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md:16]: Adding documents to a collection from different regions no longer fails when a unique key is claimed by another document.

File: docs/release-notes/ReleaseNotes_17_9.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Release Notes 17.9
+
+This update provides fixes for customer-reported issues.
+
+## Fixed Issues
+
+* API-179 - Fixed issues with API server latency.
+* STRM-296 / STRM-160 / STRM-215 - Fixed an issue where a metadata error in streams caused duplicate messages.
+* DEPL-168 / STRM-274 / STRM-255 - A timeout issue occasionally caused probe failures in Zookeeper.
```
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md:14]: Timeout issues no longer cause probe failures in Zookeeper.

File: docs/release-notes/ReleaseNotes_17_9.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 1
+title: Release Notes 17.9
+---
+
+# Release Notes 17.9
+
+This update provides fixes for customer-reported issues.
```
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md:8]: This document describes what has changed in Macrometa release 17.9

File: docs/release-notes/ReleaseNotes_17_9.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## Fixed Issues
```
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md]: Bug Fixes

File: docs/release-notes/ReleaseNotes_17_9.md (hunk 3 above)
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md]:
```suggestion
This document describes what has changed in Macrometa release 17.9.
```

Comment (diana-macrometa) [reply docs/release-notes/ReleaseNotes_17_9.md]: Let's try to use language that we can reuse for future releases. Future releases might include features, known issues, and so on as well as bug fixes.

File: docs/release-notes/ReleaseNotes_17_9.md (hunk 4 above)
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md]:
```suggestion
## Bug Fixes
```

File: docs/release-notes/ReleaseNotes_17_9.md (hunk 2 above)
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md]:
```suggestion
* DEPL-168 / STRM-274 / STRM-255 - Timeout issues no longer cause probe failures in Zookeeper.
```

File: docs/release-notes/ReleaseNotes_17_9.md (hunk 1 above)
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md]:
```suggestion
* DB-1442 - Adding documents to a collection from different regions no longer fails when a unique key is claimed by another document.
```

File: docs/release-notes/ReleaseNotes_17_9.md (hunk 3 above)
Comment (diana-macrometa) [reply docs/release-notes/ReleaseNotes_17_9.md]: Also, if we can get an official release date, that would be great to include here.

**Review by jamesm-macrometa — APPROVED** [review]
Looks good to me.

File: docs/release-notes/ReleaseNotes_17_9.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+# Release Notes 17.9
+
+This document describes what has changed in Macrometa release 17.9.
+
+## Bug Fixes
+
+* API-179 - Fixed issues with API server latency.
+* STRM-296 / STRM-160 / STRM-215 - Fixed an issue where a metadata error in streams caused duplicate messages.
+* DEPL-168 / STRM-274 / STRM-255 - A timeout issue occasionally caused probe failures in Zookeeper.
```
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md]:
```suggestion
* DEPL-168 / STRM-274 / STRM-255 - Fixed a timeout issue occasionally caused probe failures in Zookeeper.
```

**Comment by diana-macrometa** [comment]
@borisgmacrometa and @eddie-macrometa, please review the release notes and let us know if you see any issues.

File: docs/release-notes/ReleaseNotes_17_9.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 1
+title: Release Notes 17.9
+---
+
+# Release Notes 17.9
+
+This document describes what has changed in Macrometa release 17.9.
```
Comment (diana-macrometa) [inline docs/release-notes/ReleaseNotes_17_9.md]: @jamesm-macrometa - I'm told there is an API call for checking the release you're on. Could you add a link to that in this paragraph?

**Comment by diana-macrometa** [comment]
@eddie-macrometa and @borisgmacrometa, we plan to post this tomorrow. Please provide any feedback or changes before then.

@SpringNgMacrometa and @anandani-manish, FYI, this is where release notes will be living going forward.

**Pull request merged**